### ISSUE TITLE
Example dashboard

### ISSUE THREAD

**Issue by yesoreyeram** [grafana/sentry-datasource#25]
It would nice to have a built-in dashboard and few sample dashboards published to grafana.com

**Comment by eht16** [comment]
Same need here.

I'm quite new to Grafana and not yet experienced enough to build a full dashboard.
Does anyone already done it and willing to share a dashboard for this plugin?

I'm wondering about the dashboard shown in the first screenshot on https://sentry.io/integrations/grafana/. This seems to be the same as https://grafana.com/grafana/dashboards/13941-sentry-issues-events-overview/ though the latter one does not work with this plugin and seems to require Prometheus as data source.
Am I missing something?

**Comment by Azrael-git** [comment]
+1

**Comment by expcapitaldev** [comment]
any updates on this?

**Comment by BennyG93** [comment]
Agreed, it would be very nice if there was a dashboard built using the Sentry Datasource (not the prometheus exporter).

**Comment by lesaux** [comment]
Without a dashboard, this datasource is not very useful in grafana. 

It's not obvious how to produce the equivalent of 
```sum(sentry_issues_bucket{project_slug=~"$project", environment=~"$environment", le="1h"}) by (project_slug)```

with the sentry datasource.

**Comment by adean-consensys** [comment]
Also calling out anyone else willing to share their Sentry dashboards.  Examples are needed!  Thank you
